Containerfile

```diff
@@ -0,0 +1,24 @@
+ARG BASE_CONTAINER_IMAGE=registry.fedoraproject.org/fedora:latest
```

Comment:
The commit message talks about adding a Makefile, but there is now also a Containerfile, probably worth expanding the commit message a bit more.

Reply:
… enriched the commit message